ros_ign_dev/ign_gazebo_dev/cmake/ignition_gazebo_dev-extras.cmake

```diff
@@ -0,0 +1,17 @@
+# This code here calls and sets required igniton gazebo based on ROS Distribution so,packages can directly go find_package(ign_gazebo_dev) so,that 
+# they don't need to change code with different ros distros
+if(DEFINED ENV{ROS_DISTRO})    
```

Comment:
In order to use the `ROS_DISTRO` environment variable in CMake configurations this package should have a `build_depend` on the `ros_environment` package which provides hooks to set `ROS_DISTRO`, `ROS_VERSION`, and `ROS_PYTHON_VERSION`.

Reply:
I have added ros_environment dependency.